src/config.ts

```diff
@@ -76,6 +76,7 @@ export const LockupAbledPoolIds: {
 	'670': true,
 	'678': true,
 	'679': true,
+	'691': true,
```

Comment:
Can we please put the External incentive gauge in a different PR?

Reply:
Sure thing, removed externals from this PR.